### ISSUE TITLE
Package missing

### ISSUE THREAD

**Issue by Agazoth** [Microsoft/Partner-Center-PowerShell#77]
# Steps to reproduce

When running Connect-PartnerCenter on a vanilla installation of Powershell Core, the following error occurs:
```powerwhell
Connect-PartnerCenter : Could not load file or assembly 'Microsoft.IdentityModel.Clients.ActiveDirectory, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. Could not find or load a specific file. (Exception from HRESULT: 0x80131621)
At line:1 char:1
+ Connect-PartnerCenter -AccessToken $pcToken.AccessToken -ApplicationI ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Connect-PartnerCenter], FileLoadException
+ FullyQualifiedErrorId : System.IO.FileLoadException,Microsoft.Store.PartnerCenter.PowerShell.Commands.ConnectPartnerCenter
```

## Expected behavior
Should load the module and run the command

If the package is not installed, an error should state, that the module could not be loaded due to missing requirements.

## Actual behavior

Throws an error

## Diagnostic logs

Will be shared if needed

## Environment

```powershell
PS C:\> get-package Microsoft.IdentityModel.Clients.ActiveDirectory
get-package : No package found for 'Microsoft.IdentityModel.Clients.ActiveDirectory'.
At line:1 char:1
+ get-package Microsoft.IdentityModel.Clients.ActiveDirectory
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (Microsoft.Power...lets.GetPackage:GetPackage) [Get-Package], Exception
+ FullyQualifiedErrorId : NoMatchFound,Microsoft.PowerShell.PackageManagement.Cmdlets.GetPackage

PS C:\> find-package Microsoft.IdentityModel.Clients.ActiveDirectory

Name                           Version          Source           Summary
----                           -------          ------           -------
Microsoft.IdentityModel.Cli... 4.5.1            nuget.org        This package contains the binaries of the Active Di...

PS C:\> $host

Name             : ConsoleHost
Version          : 6.2.0-preview.3
InstanceId       : f7acc933-b8aa-4f45-bf06-058b5bfe783f
UI               : System.Management.Automation.Internal.Host.InternalHostUserInterface
CurrentCulture   : en-US
CurrentUICulture : en-US
PrivateData      : Microsoft.PowerShell.ConsoleHost+ConsoleColorProxy
DebuggerEnabled  : True
IsRunspacePushed : False
Runspace         : System.Management.Automation.Runspaces.LocalRunspace
```

Even after  installing the package the error occours

```powershell
Connect-PartnerCenter : Could not load file or assembly 'Microsoft.IdentityModel.Clients.ActiveDirectory, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. Could not find or load a specific file. (Exception from HRESULT: 0x80131621)
At line:1 char:1
+ Connect-PartnerCenter -AccessToken $pcToken.AccessToken -ApplicationI ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Connect-PartnerCenter], FileLoadException
+ FullyQualifiedErrorId : System.IO.FileLoadException,Microsoft.Store.PartnerCenter.PowerShell.Commands.ConnectPartnerCenter

PS C:\Users\AxelBøgAndersen> Get-Package Microsoft.IdentityModel.Clients.ActiveDirectory

Name                           Version          Source                           ProviderName
----                           -------          ------                           ------------
Microsoft.IdentityModel.Cli... 4.5.0            C:\Program Files\PackageManag... NuGet
```

**Comment by isaiahwilliams** [comment]
Hi @Agazoth, 

We will have this issue resolved with the release of 1.5.1902.4. Our hope is to have this version available before the end of the week. Please let us know if you have any other issues.

**Comment by isaiahwilliams** [comment]
Hi @Agazoth, 

We just released an update that resolves this issue. Please let us know if you have any other issues.
